src/main/java/com/sun/tdk/signaturetest/core/MemberCollectionBuilder.java

```diff
@@ -76,6 +76,16 @@
      */
     static Logger logger = Logger.getLogger(MemberCollectionBuilder.class.getName());
 
+    public MemberCollectionBuilder(Log log, JDKExclude jdkExclude) {
+        this.cc = new ClassCorrector(log, jdkExclude);
```

Comment:
This can be 

this.cc = jdkExclude == null ? new ClassCorrector(log) : new ClassCorrector(log, jdkExclude);

again related to avoiding NPEs